monorepo-builder.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
+  package_directories:
+    - 'src'
```

Comment:
Since these are now _third-party_ code, perhaps they could live in `packages` instead of `src`?

Reply:
I'm not sure what you mean by "third-party code". I have no strong preferences about the directory name.

Reply:
I might be misunderstanding it - Is this thought correct?

- I clone a fresh Square One project
- `plugins/core/src` folder contains only code related to that specific project
- I require a package from the monorepo
- Now `plugins/core/src` folder also contains that monorepo package, which is tribe lib's code, not the project's

Reply:
Nope. `plugins/core/src` still contains the project code. The vendor code (i.e., what you install from our mini-repos) goes to `vendor/moderntribe/square1-*`

Reply:
:aw-yeah: thanks for the clarification